google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -20,6 +20,8 @@
 
 /** Options for the session pool used by {@code DatabaseClient}. */
 public class SessionPoolOptions {
+  private static final int DEFAULT_MAX_SESSIONS = 2000;
```

Comment:
As discussed separately, this is higher than necessary given the restriction on number of outstanding RPCs per channel and a default of 4 channels. Shall we make it 400?